migrations/20260713150000-backfill-meme-card-drop-mappings.js

```
@@ -0,0 +1,103 @@
+'use strict';
+
+function parsePositiveInteger(value, label) {
+  var parsed = Number(value);
+  if (!Number.isSafeInteger(parsed) || parsed < 1) {
+    throw new Error('Invalid ' + label + ': ' + value);
+  }
+  return parsed;
+}
+
+exports.up = async function(db) {
+  var mainStageWaveId = process.env.MAIN_STAGE_WAVE_ID;
```

ESLint

[error] 'process' is not defined.

(no-undef)